### PR TITLE
Fixing grammar in readme.md

### PR DESCRIPTION
**Pull request by oralekin**
Fixes two grammar issues, couldn't find a way to get rid of those pesky [here](www.placeholder.de.org.net/hi)'s.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
 
 Contributions to the osu!wiki are now done through GitHub.
 
-A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [here](/wiki/osu!wiki_contribution_guide).
+A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [](/wiki/osu!wiki_contribution_guide).
```
Comment (MegaApplePi) [inline README.md]: This will remove the link overall!
Use `[osu!wiki contribution guide](/wiki/osu!wiki_contribution_guide).`

File: README.md [hunk 2]
```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
...
 
 The article style guide, which _must_ be adhered to for all new and rewritten pages going forward, is available [here](/wiki/Article_Style_Guide).
```
Comment (MegaApplePi) [inline README.md]: Why not fix this too while you were at it...? replace with `[Article Style Guide](/wiki/Article_Style_Guide).`

File: README.md [hunk 3]
```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
 
 Contributions to the osu!wiki are now done through GitHub.
 
-A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [here](/wiki/osu!wiki_contribution_guide).
+A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [Article Style Guide](/wiki/Article_Style_Guide)..
```
Comment (Shadowtail117) [inline README.md]: Shouldn't you just merge lines 25 and 27 into one; something like "A basic guide to using Github for the osu!wiki project that _must_ be adhered to for all new and rewritten pages going forward is available here: [link]." Having two separate lines leading to the same folder seems redundant.

File: README.md [hunk 4]
```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
 
 Contributions to the osu!wiki are now done through GitHub.
 
-A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [here](/wiki/osu!wiki_contribution_guide).
+A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [osu!wiki Contribution Guide](/wiki/osu!wiki_contribution_guide).
```
Comment (TPGPL) [inline README.md]: `is available **in** [osu!wiki Contribution Guide](/wiki/osu!wiki_contribution_guide).`

File: README.md [hunk 5]
```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
...
 
-The article style guide, which _must_ be adhered to for all new and rewritten pages going forward, is available [here](/wiki/Article_Style_Guide).
+The article style guide, which _must_ be adhered to for all new and rewritten pages going forward, is available [Article Style Guide](/wiki/Article_Style_Guide).
```
Comment (TPGPL) [inline README.md]: same here

File: README.md [hunk 6]
```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
 
 Contributions to the osu!wiki are now done through GitHub.
 
-A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [here](/wiki/osu!wiki_contribution_guide).
+A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available at [osu!wiki Contribution Guide](/wiki/osu!wiki_contribution_guide).
```
Comment (Shadowtail117) [inline README.md]: This is just my pedantry speaking, but shouldn't it be "...is available at the osu!wiki Contribution Guide"?

File: README.md [hunk 7]
```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
...
 
-The article style guide, which _must_ be adhered to for all new and rewritten pages going forward, is available [here](/wiki/Article_Style_Guide).
+The article style guide, which _must_ be adhered to for all new and rewritten pages going forward, is available at [Article Style Guide](/wiki/Article_Style_Guide).
```
Comment (Shadowtail117) [inline README.md]: Same as above.

File: README.md (hunk 6 above)
Comment (oralekin) [inline README.md]: I agree, so did TGPGL apparently...

**Comment by sr229** [comment]
@Replfez this PR is ready to merge. Awaiting your review

**Comment by oralekin** [comment]
sr229 meant @Repflez

**Pull request merged**